src/ast-types/smdast/index.ts

```diff
@@ -16,12 +17,18 @@ export {
   IParagraph,
   ILink,
   ILinkReference,
+  IYAML,
   IImage,
   IImageReference,
   IInlineCode,
   ITextNode,
   IList,
   IListItem,
+  IFootnote,
+  IFootnoteReference,
+  IFootnoteDefinition,
+  IElement,
```

Comment:
`IElement` is probably not needed here, `./element` exports a different `IElement` definition.